.github/workflows/deploy-examples.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy example apps to Fly
+
+on:
+  # Allow calling this workflow from other workflows
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+    secrets:
+      FLY_API_TOKEN:
+        description: "Fly API token for deploying apps"
+        required: true
+
+  # Allow calling this workflow manually from the GitHub UI
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+
+  # TODO: REMOVE THIS BEFORE MERGING, JUST FOR TESTING
+  push:
+    branches:
+      - 2831-automatically-deploy-all-example-apps-on-release
+
+concurrency:
+  group: "deploy-examples"
+  cancel-in-progress: true
```

Comment:
I'm guessing this tells github to run all jobs in the group in parallel.

But where do we tell it what's in the group. In the file name, or is the `concurrency` dictionary enough?

Reply:
> this tells github to run all jobs in the group in parallel

No. This is about _workflow runs_, and not _jobs_ inside those runs. What this means is that, if a new run of this workflow is queued, this run (and both jobs inside it) will get immediately cancelled and run the newer one.

> what's in the group

Everything with the same `concurrency.group` value is in the same group.